sqlglot/lineage.py

```diff
@@ -18,6 +18,7 @@ class Node:
     expression: exp.Expression
     source: exp.Expression
     downstream: t.List[Node] = field(default_factory=list)
+    source_name: str = ""
```

Comment:
what's the difference between alias and source_name ?

Reply:
`source_name` is intended to be the name of the source (the query passed into the `sources` argument of the lineage function) this node came from. `alias` is intended for the CTE name the node came from.